src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md

```diff
@@ -29,11 +29,10 @@ that not all uses of `wasm32-unknown-unknown` are using JavaScript and the web.
 
 ## Target maintainers
 
-When this target was added to the compiler platform-specific documentation here
+When this target was added to the compiler, platform-specific documentation here
 was not maintained at that time. This means that the list below is not
```

Comment:
```suggestion
At the time this target was added to the compiler, platform-specific documentation here
was not maintained. This means that the list below is not
```
I belive this is a 'more easy to read' version?

Reply:
yeah, that is an improvement